### PR TITLE
Fix issues introduced in PR #636

### PR DESCRIPTION
**Pull request by asofron**
[LISAv2 Test Results Summary]
Test Run On           : 01/23/2019 06:59:53
VHD Under Test        : ubuntu_18.04.1.vhdx
Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
Total Time (dd:hh:mm) : 0:0:16

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
-------------------------------------------------------------------------------------------------------------------
--
    1 STRESSTEST-CHANGE-MTU-RELOAD-NETVSC                                               PASS                14.44 

[LISAv2 Test Results Summary]
Test Run On           : 01/23/2019 07:21:33
VHD Under Test        : Centos_7.5_x64.vhdx
Total Test Cases      : 1 (1 Pass, 0 Fail, 0 Abort)
Total Time (dd:hh:mm) : 0:0:16

   ID TestCaseName                                                                TestResult TestDuration(in minutes) 
---------------------------------------------------------------------------------------------------------------------
    1 STRESSTEST-CHANGE-MTU-RELOAD-NETVSC                                               PASS                 14.4

### CODE REVIEW

File: Libraries/IntegrationServiceLibrary.psm1 [hunk 1]
```diff
@@ -279,10 +279,9 @@ function Get-IPv4AndWaitForSSHStart {
 	}
 
 	# Cache fingerprint, Check ssh is functional after reboot
-	$null = Write-Output "yes" | .\Tools\plink.exe -C -pw $Password -P $VmPort $User@$new_ip 'exit 0'
-	Write-LogInfo ".\Tools\plink.exe -C -pw $Password -P $VmPort $User@$new_ip"
+	Write-Output "yes" | .\Tools\plink.exe -C -pw $Password -P $VmPort $User@$new_ip 'exit 0'
 	$TestConnection = .\Tools\plink.exe -C -pw $Password -P $VmPort $User@$new_ip "echo Connected"
-	if (-not ($TestConnection)) {
+	if ($TestConnection -ne "Connected") {
```
Comment (ader1990) [inline Libraries/IntegrationServiceLibrary.psm1]: remove the else as it was before

File: Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1 [hunk 2]
```diff
@@ -41,8 +41,7 @@ ifdown eth0 && ifup eth0
             -command "sleep 5 && ip link set dev eth0 mtu $i" -RunAsSudo
 
         Start-Sleep -s 30
-        $TestConnection = Test-Connection -ComputerName $ipv4
-        if (-not $TestConnection) {
+        if (-not $( Test-Connection -ComputerName $ipv4)) {
```
Comment (ader1990) [inline Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1]: leave the code as before, this code does not work as expected

Comment (asofron) [reply Testscripts/Windows/STRESS-MTU-Netvsc-Reload.ps1]: Done

File: Libraries/IntegrationServiceLibrary.psm1 (hunk 1 above)
Comment (asofron) [reply Libraries/IntegrationServiceLibrary.psm1]: Fixed added Out-null for output

**Pull request merged**